profile/README.md

```diff
@@ -0,0 +1,29 @@
+##  Welcome to the Keptn project 👋
+
+[![Website](https://img.shields.io/static/v1?label=Website&message=keptn.sh&color=blue)](https://keptn.sh/)
+[![Roadmap](https://img.shields.io/static/v1?label=Roadmap&message=public&color=green)](https://github.com/orgs/keptn/projects/1)
+[![Governance](https://img.shields.io/static/v1?label=Governance&message=process&color=yellow)](https://github.com/keptn/keptn/blob/master/GOVERNANCE.md)
+[![Participate](https://img.shields.io/static/v1?label=Contributing&message=guide&color=blue)](https://keptn.sh/community/)
+
+Keptn is an event-driven orchestration engine for cloud-native apps.
+Started in 2018,
+It uses a declarative approach to build scalable automation
+for delivery and operations which can be scaled to a large number of services.
+Keptn evaluates _Service Level Indicators (SLOs)_ and provides a dashboard, alerts, and auto-remediation for them.
+It also provides [integrations](https://keptn.sh/docs/integrations/)
+with the Cloud Native ecosystem 
+including _Prometheus_, _ArgoCD_, _CloudEvents_, _Backstage_,
+and many other cloud-native projects.
+
+Keptn a member project of the [Cloud Native Computing Foundation](https://cncf.io).
+
+**Want to contribute?**
+There is a big community behind Keptn.
+We invite you to join us and participate. 
+Connect to the Community on
```

Comment:
```suggestion
Connect to the community on
```
Since "community" is not capitalized in line 21, seems odd to capitalize it here.